src/components/Metadata.vue

```diff
@@ -352,7 +352,10 @@ export default defineComponent({
     },
 
     people(): IFace[] {
-      return this.baseInfo?.clusters?.recognize ?? [];
+      if (this.routeIsFaceRecognition)
```

Comment:
There's a missing case here: if recognize is disabled then we want to use facerecognition by default